### ISSUE TITLE
Compile data - Catch

### ISSUE THREAD

**Issue by brianlangseth-NOAA** [nwfsc-assess/canary_2023#9]
Start compiling necessary data:

1. Catches
 - Recent
   - Obtain from PacFIN and RecFIN
    - For WA do we get these 
   - Are data updated onto these up to 2022?
 - Historical
  - Where to obtain
  - Is WA catches separated by coastal US, coastal Canadian, Puget sound?

**Comment by brianlangseth-NOAA** [comment]
@chantelwetzel-noaa placed canary PacFIN data (as of Sept 1) on the FRAM network:
Assessments\Assessment Data\2023 Assessment Cycle\canary rockfish\PacFIN data

Thanks Chantel

**Comment by okenk** [comment]
Per meeting with ODFW, determine PacFIN gear codes for the different fleets

**Comment by brianlangseth-NOAA** [comment]
@aliwhitman Here are the pacfin codes, divided into trawl and non-trawl as I see them, for oregon data with the species name containing some version of CANARY. I need to check if they align reasonably with the last assessment

Trawl
392 bottom trawl - mud(sole) gear    
391 bottom trawl - roller gear                      
390 GROUNDFISH TRAWL                 
360 MIDWATER TRAWL                   
393 SELECTIVE FLATFISH TRAWL             
380 SHRIMP TRAWL (DR)             
370 SHRIMP TRAWL (SR) 

Non-trawl
400 CRAB POT 
300 FISH POT              
340 HOOK AND LINE 
350 LONGLINE                
120 OCEAN TROLL 

Is the midwater trawl gear actually the at-sea hake fishery?

**Comment by brianlangseth-NOAA** [comment]
Found a file from the 2015 assessment that lists oregon non-trawl gears. These are based on older codes: CPT	DST	FPT	LGL	OHL	SHT	SST	TRL. 

I can figure out most, but @aliwhitman do you know what DST, SHT, and SST would be? Would DST and SST be the two shrimp trawls?

**Comment by chantelwetzel-noaa** [comment]
The PacFIN gear codes can be found online [here](https://pacfin.psmfc.org/pacfin_pub/data_rpts_pub/code_lists/agency_gears.txt). Your inclination is correct that they all pertain to different shrimp gear.

**Comment by brianlangseth-NOAA** [comment]
Older PcFIN gear codes can be found [here](https://pacfin.psmfc.org/pacfin_pub/data_rpts_pub/code_lists/gr.txt). Previous assessment's .dmp files had a few that aren't in the link in the comment above

**Comment by brianlangseth-NOAA** [comment]
@aliwhitman I was able track down the codes used in the 2015 assessment based on pacfin_gear_codes or at the time GRID codes (thanks to @chantelwetzel-noaa).

**Trawl**
c("BMT","BTT","CBF","CBJ","DNT","FFT","FTS","GFL","GFS","GFT","LFZ","MDT","MPT","OTW","PRT","RLT","SFZ","SRM","TWL")] <- "TWL"

**Non-trawl**
c("DST", "PWT","SHT","SST", "TWS")] <- "TWS" Shrimp Trawls
c("DRL","HDL","HLR","JIG","LGL","OHL","POL","STL","VHL","HKL")] <- "HKL" Hook and Line
c("DGN","DPN","GLN","ONT","SEN","SGN","STN","TML","NET")] <- "NET" Net gears
c("CLP","CPT","FPT","LPT","OPT","PRW","SPT","POT")] <- "POT" Pot gears
c("BTR", "DVG", "TRL")] <- "OTH" Other - mostly troll

Thus shrimp trawls were included with non-trawl gear, and midwater trawl was included in trawl. Do think we should keep these designations? 

If we alter we will ahve to include in the bridging analysis. Unless you are fine with what was done previously, an alternative is to provide 1) non-trawl without shrimp, 2) shrimp, 3) trawl without midwater, and 4) midwater so as to tease apart any issues and add as we see fit.

**Comment by aliwhitman** [comment]
After checking with other folks at ODFW, I would probably advise that shrimp gear be lumped with trawl gear. I know that it's very minor for canary but it really is more similar to other trawl gear than non-trawl from a selectivity standpoint.  

Also, I am confused about why the mid-water trawl was lumped with trawl gear if we had a separate hake fleet? Perhaps @chantelwetzel-noaa could speak to this.

**Comment by brianlangseth-NOAA** [comment]
@aliwhitman Confirmed about shrimp trawl. As for midwater, perhaps looking at chantel's responses in issue #18 is the answer you are looking for?

**Comment by aliwhitman** [comment]
Yes, thank you for steering me to the ASHOP thread, that makes sense to continue to retain mid-water trawl with the trawl fleet. Thanks!!

**Comment by brianlangseth-NOAA** [comment]
@JohnBudrick, do you see an issue with the above gear codes aggregations for CA, and whether you'd also think shrimp trawls should be lumped together with trawl gears? In the 2015, shrimp trawl was listed with non-trawl.

**Comment by brianlangseth-NOAA** [comment]
RecFIN pulls from public site are on google drive folder under "Data > Brian RecFIN pulls" and the private access 506 data pull is in the same location but in the "_CONFIDENTIAL" subfolder

**Comment by brianlangseth-NOAA** [comment]
@okenk Do you remember where this was specified. Want to document here so we remeber, and would like to include original "source". 

California recfin catches need for 2020 need are missing for April - July due to Covid. California has data to extrapolate catch amounts for that time period, which we would add to the rest of catches for that year.

**Comment by okenk** [comment]
@JohnBudrick, didn't you tells us that CA did not do any rec sampling April-July, 2020?

**Comment by JohnBudrick** [comment]
Hi Kiva,

It came up in one of our group discussions.  I will follow up internally about proxies used in the past by the GMT at the time.

Thanks,

John

**Comment by aliwhitman** [comment]
With regards to the Oregon historic commercial catch time series, we have discovered that during the last assessment for black rockfish, trawl species comps were determined to be too variable from year to year, and so they were aggregated to stabilize the historic trawl landings. We are likely going to be using the same methodology for this assessment. For the last Canary assessment, no changes were made to the historic time series, despite being based on the same data. 

Here's a snapshot of our species comps for Canary during the period in question. These are annually aggregated comps (which is all we have, unfortunately, the raw comps are no longer available). Canary was one of the primary trawl species at the time, which gives me more confidence in the species comps than it does for Black.  (Ignore 1C and 3B below, they're outside of OR).
![image](https://user-images.githubusercontent.com/80843508/213246497-678f9829-7d16-417d-96b4-6d74f880e3de.png)

I wanted to make you guys aware of this, but ultimately, I would still recommend that we stick with the historic time series (identical to the last Canary assessment) but include a recommendation to revisit the historic trawl landings in Oregon (we are planning to include a similar recommendation in the Black rockfish assessment) so that this could get sorted out in a comprehensive fashion.  What are your thoughts on this?

**Comment by okenk** [comment]
Thanks so much for looking into this Ali. I am happy to defer to your judgement, especially if that big green spike is not relevant! :)

**Comment by brianlangseth-NOAA** [comment]
Documenting our choices here: Per our weekly gatherings we've decided to use the historical expansion for OR commercial as is for our base assumptions, that is not breaking out the portions potentially caught in WA waters. The reason for this is that canary is coastwide (as opposed to black rockfish for where this issue first appeared), as well as that there is likely part of WA landings caught in OR waters that we would not be applying. 

Explorations around allocating some landings in OR to catches in WA can be done within a sensitivity, alongside choices around the historical CA landings caught in OR/WA (issue #51), using rough estimates to assess the effect of that uncertainty. As @aliwhitman mentions above, a more comprehensive manner of resolving this issue is best for doing out of cycle, thus this will be a future research request.
